### PR TITLE
Prow version auto bumper part 4: README and images

### PR DESCRIPTION
**Pull request by chaodaiG**
Part 4 of Prow version auto bumper tool:
- Add README
- Add dockerfile
- Cleanup comments

### CODE REVIEW

File: tools/prow-auto-bumper/pullrequest.go [hunk 1]
```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// pullrequest.go and creates git commit and Pull Requests
```
Comment (Fredy-Z) [inline tools/prow-auto-bumper/pullrequest.go]: Typo?

File: tools/prow-auto-bumper/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# prow-auto-bumper
+
+prow-auto-bumper is a tool that identifies recent stable Prow version in k8s/test-infra repo, updates knative/test-infra to use this new version, and creates Pull Request.
+
+## Basic Usage
+
+Flags for this tool are:
+
+* `--github-account` [Required] specifies the path of file containing Github token for Github API calls.
+* `--git-userid` [Required] specifies the github ID of user for hosting fork, i.e. Github ID of bot.
```
Comment (Fredy-Z) [inline tools/prow-auto-bumper/README.md]: `Github`/`github` -> `GitHub`?

**Comment by Fredy-Z** [comment]
/hold
If you want to solve a few nits.

File: tools/prow-auto-bumper/README.md (hunk 2 above)
Comment (chaodaiG) [reply tools/prow-auto-bumper/README.md]: done

File: tools/prow-auto-bumper/pullrequest.go (hunk 1 above)
Comment (chaodaiG) [reply tools/prow-auto-bumper/pullrequest.go]: removed and

**Comment by chaodaiG** [comment]
updated, a new lgtm is needed @Fredy-Z

**Comment by Fredy-Z** [comment]
Thanks!
/hold cancel
/lgtm

**Pull request merged**